apps/speed.c

```diff
@@ -1466,6 +1486,24 @@ int speed_main(int argc, char **argv)
             if (!opt_int(opt_arg(), &primes))
                 goto end;
             break;
+        case OPT_SECONDS:
+            seconds.sym = atoi(opt_arg());
+            seconds.rsa = atoi(opt_arg());
+            seconds.dsa = atoi(opt_arg());
+            seconds.ecdsa = atoi(opt_arg());
+            seconds.ecdh = atoi(opt_arg());
```

Comment:
Even though it's as far from performance-critical path as only possible, I'd say that calling single call to `atoi(opt_arg())` would still be more appropriate. It's more readable that way, as you don't have to keep in mind how `opt_arg()` looks like. `seconds.sym = seconds.rsa = ... atoi(opt_arg())` would do :-)

Reply:
agreed. fixed.